Jenkinsfile

```diff
@@ -268,11 +244,8 @@ timestamps {
 
 		// Run unit tests in parallel for android/iOS
 		stage('Test') {
-			parallel(
-				'android unit tests': unitTests('android', nodeVersion, npmVersion, targetBranch),
-				'iOS unit tests': unitTests('ios', nodeVersion, npmVersion, targetBranch),
-				failFast: true
-			)
+			unitTests('android', nodeVersion, npmVersion, targetBranch),
```

Comment:
why not in parallel? you can set failFast: false if you want both to run to completion even if one of them fails... (so you can try and fix issues in either/both each round of commits 😄 )